src/main/java/roomescape/dao/reservation/ReservationDao.java

```diff
@@ -0,0 +1,14 @@
+package roomescape.dao.reservation;
+
+import java.util.List;
+import roomescape.domain.Reservation;
+import roomescape.dto.request.ReservationCreateRequest;
+
+public interface ReservationDao {
+
+    List<Reservation> findAll();
+
+    long create(ReservationCreateRequest reservationCreateRequest);
+
+    void delete(final Long id);
```

Comment:
```suggestion
    void delete(long id);
```
1. 인터페이스에는 final이 없어도 되겠네요.
2. id를 무조건 받아야할건데 nullable long을 써도 될까요?

Reply:
2번에 있어서 학습하였습니다.

원시값 - null을 고려해야 하는 상황, 즉 요청 바디 DTO처럼 값이 안들어올 수 있는 경우, id 같은 PK에 없음을 표시하기 위해 (원시는 null일 경우 0으로 계산)

객체값 - null을 고려하지 않아도 되는 상황, 즉 값이 무조건 들어오는 경우 PathVariable은 값이 어떠든 결국 uri에 표시되어야 요청이 들어오기 때문에 null일 필요 없다. findById, Response 등 특이 사항이 없는 경우


//TODO 커밋 로그